core/src/avm2/globals/flash/events/NativeWindowBoundsEvent.as

```diff
@@ -0,0 +1,52 @@
+// The initial version of this file was autogenerated from the official AS3 reference at
+// https://help.adobe.com/en_US/FlashPlatform/reference/actionscript/3/flash/events/NativeWindowBoundsEvent.html
+// by https://github.com/golfinq/ActionScript_Event_Builder
+// It won't be regenerated in the future, so feel free to edit and/or fix
+
+package flash.events
+{
+
+  import flash.geom.Rectangle;
+
+  [API("661")]
+  public class NativeWindowBoundsEvent extends Event
+  {
+    public static const RESIZE:String = "resize";
```

Comment:
There are 4 constants in the docs, only one of them here:
 - `MOVING`
 - `MOVE`
 - `RESIZING`
 - `RESIZE`